frontend/packages/component-library/src/dropdown/CustomDropdown.tsx

```diff
@@ -263,4 +277,10 @@ const CustomDropdown: React.FunctionComponent<CustomDropdownProps> = ({
   );
 };
 
-export default CustomDropdown;
+const WrappedCustomDropdown = (props: CustomDropdownProps) => (
```

Comment:
This is added to `CustomDropdown` from callers because it is duplicated in each call of `CustomDropdown` and `CustomDropdown` does not show anything when it is missing.